src/EncounterPage/EnemyEntry.js

```diff
@@ -1,18 +1,29 @@
-import React from 'react'
+import React, { useState } from 'react'
 import styled from 'styled-components/macro'
 import PropTypes from 'prop-types'
+import HPCalculator from './HPCalculator'
 
 EnemyEntry.propTypes = {
   enemy: PropTypes.string.isRequired,
   ac: PropTypes.number.isRequired,
+  HP: PropTypes.number.isRequired,
   maxHP: PropTypes.number.isRequired,
 }
 
-export default function EnemyEntry({ enemy, ac, maxHP }) {
+export default function EnemyEntry({ enemy, ac, HP, maxHP }) {
+  const [currentHP, setCurrentHP] = useState(Number(HP))
+
   return (
     <EnemyEntryStyled>
       <h2>{enemy}</h2>
-      <p>HP: {maxHP}</p>
+      <p>
+        HP: {currentHP > 0 ? currentHP : 0}/{maxHP}
```

Comment:
If I understand this part correctly currentHP could be negative, but you don't want to display negative numbers. I think it would be a better idea to prevent currentHP to becoming negative in the first place. Otherwise you might get some weird calculating behavior if the shown value differs from the value in the state.